src/content/en/2022/performance.md

```diff
@@ -1,22 +1,989 @@
+
+#### INP by rank
+
+To see how unevenly distributed INP performance is across the web, it's useful to segment websites by their popularity ranking.
+
+{{ figure_markup(
+  image="inp-performance-by-rank.png",
+  caption="INP performance by rank.",
+  description="Stacked bar chart showing INP performance on mobile devices segmented by rank. The top 1,000 sites have 27% of good INP experiences, 52% have needs improvement INP experiences, and 20% have poor INP experiences. For the top 10,000 sites it's 25%, 50%, and 24% respectively, for the top 100,000 sites it's 31%, 50%, and 18%, for the top 1,000,000 sites it's 42%, 46%, and 12%, and finally for all sites it's 55% of good, 27% of needs improvement, and 5% of poor experiences.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vR-dJP3uphZoGE5A_luniNBFm5V2ww6irfOxANg0hrMid7gjgrtchsN_utOIDOvVZUjIwpmUBb27nHF/pubchart?oid=1730743352&format=interactive",
+  sheets_gid="805166525",
+  sql_file="web_vitals_by_rank_and_device.sql"
+  )
+}}
+
+27% of the top 1k most popular websites have good mobile INP. As the site popularity decreases, the percent having good mobile INP does something funny; it worsens at bit at the top 10k rank to 25%, then it improves to 31% at the top 100k, 41% at the top million, and it ultimately lands at 55% for all websites. Except for the top 1k, it seems that INP performance is inversely proportional to site popularity.
+
+{{ figure_markup(
+  image="js-bytes-rank.png",
+  caption="Median amount of JavaScript loaded per page, by rank.",
+  description="Bar chart showing the median kilobytes of JavaScript per page, grouped by device and site rank. The median top 1 thousand mobile page loads 604 KB of JavaScript, 680 in the top 10 thousand, 610 in the top 100 thousand, 583 in the top million, and 462 overall. The values for desktop follow the same trend and are larger by tens of kilobytes.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vR-dJP3uphZoGE5A_luniNBFm5V2ww6irfOxANg0hrMid7gjgrtchsN_utOIDOvVZUjIwpmUBb27nHF/pubchart?oid=1725756250&format=interactive",
+  sheets_gid="1423103831",
+  sql_file="js_bytes_rank.sql"
+  )
+}}
+
+When we look at the amount of JavaScript that the median mobile page loads for each of these ranks, it follows the same funny pattern! The median mobile page in the top 1k loads 604 KB of JavaScript, then it increases to 680 KB for the top 10k before dropping all the way down to 462 KB over all websites. These results don't prove that loading—and using—more JavaScript necessarily causes poor INP, but it definitely suggests a correlation exists.
+
+#### INP as a hypothetical CWV metric
```

Comment:
I don't believe we're supposed to use CWV as an initialism. I'd change all instances of this to "Core Web Vitals".

Reply:
Is that convention more for the official docs, or any post referencing CWV? I think given that we've used CWV in previous years' chapters and the community should easily recognize it, it's ok to keep, but not set on it.

Reply:
Resolving for now to unblock publishing, but happy to take another look afterwards if needed